threefive/stream.py

```diff
@@ -144,7 +144,8 @@ def _mk_packet_data(self, pid):
         packet_data["pid"] = pid
         prgm = self._pid_prog[pid]
         packet_data["program"] = prgm
-        packet_data["pts"] = round(self._prog_pts[prgm], 6)
+        if prgm in self._prog_pts:
```

Comment:
I didn't know you could check for key directly on a dict,
I  have been checking for keys against dict.keys(),
this is cool.